kore/src/Kore/Builtin/Verifiers.hs

```diff
@@ -181,8 +181,8 @@ lookupApplicationVerifier symbol verifiers = do
             . symbolAttributes
     getKlabel =
         fmap KlabelSymbolKey
-            . Attribute.Symbol.getKlabel
-            . Attribute.Symbol.klabel
+            . Attribute.Symbol.getSymbol
+            . Attribute.Symbol.symbolKywd
```

Comment:
This should probably be renamed to `getSymbol'Kywd` to avoid confusion

Reply:
Probably the line above that says
```
    key <- getHook symbol <|> getKlabel symbol
```
could be changed to 
```
    key <- getHook symbol <|> getKlabel symbol <|> getSymbol'Kywd symbol
```
to achieve backward compatibility.
Needs to be tested but could avoid creating a broken release.